### PR TITLE
Support forkjoinpool plugin in JDK11

### PR DESCRIPTION
**Pull request by 786991884**
fix https://github.com/apache/skywalking/issues/11633

### CODE REVIEW

**Comment by wu-sheng** [comment]
You need to add the test scenario to run in JDK 11, otherwise, this is only a manual test

**Comment by wu-sheng** [comment]
Also, you removed the pull request template, we need that, and you missed the update in the change log.

**Comment by wu-sheng** [comment]
> You need to add the test scenario to run in JDK 11, otherwise, this is only a manual test

This is still missed.

**Comment by wu-sheng** [comment]
Why do you open a new test3 with so many cases?

File: .github/workflows/plugins-jdk11-test.3.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  test:
+    needs: [ build ]
+    name: ${{ matrix.case }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    strategy:
+      matrix:
+        case:
+          - jdk11-forkjoinpool-scenario
```
Comment (wu-sheng) [inline .github/workflows/plugins-jdk11-test.3.yaml:55]: Do we need a new case? I think this is rt.jar(jdk level) change, even the codes are compiled as 1.8, the runtime method is still going to be forced to run in 11, right?

Comment (wu-sheng) [reply .github/workflows/plugins-jdk11-test.3.yaml:55]: Could we not copy the test scenario, but directly write the existing test in jdk11. Was that failing before and running now?

File: apm-sniffer/bootstrap-plugins/jdk-forkjoinpool-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdk/forkjoinpool/define/ForkJoinWorkerQueueInstrumentation.java [hunk 2]
```diff
@@ -33,6 +33,9 @@ public class ForkJoinWorkerQueueInstrumentation extends ClassInstanceMethodsEnha
     private static final String FORK_JOIN_WORKER_QUEUE_CLASS = "java.util.concurrent.ForkJoinPool$WorkQueue";
 
     private static final String FORK_JOIN_WORKER_QUEUE_RUN_TASK_METHOD = "runTask";
+    
+    private static final String FORK_JOIN_WORKER_QUEUE_RUN_TASK_METHOD_JDK11 = "topLevelExec";
```
Comment (wu-sheng) [inline apm-sniffer/bootstrap-plugins/jdk-forkjoinpool-plugin/src/main/java/org/apache/skywalking/apm/plugin/jdk/forkjoinpool/define/ForkJoinWorkerQueueInstrumentation.java:37]: Please add comments about why there are two methods to intercept.

**Comment by wu-sheng** [comment]
From [test logs](https://github.com/apache/skywalking-java/actions/runs/7162355124/job/19504471071?pr=656), your interceptor doesn't run as expected. Still, the old method existed and being intercepted.

**Comment by wu-sheng** [comment]
Tests passed, could you check this? https://github.com/apache/skywalking-java/pull/656#discussion_r1422039367

File: .github/workflows/plugins-jdk11-test.3.yaml (hunk 1 above)
Comment (786991884) [reply .github/workflows/plugins-jdk11-test.3.yaml:57]: yes, you are right. I will optimize it

**Comment by wu-sheng** [comment]
Is there any update here?

**Comment by wu-sheng** [comment]
@786991884 Do you have any update?

**Comment by wu-sheng** [comment]
@786991884 If you will be back, I hope this extra test can be removed by reusing the existing one.

I am going to merge this first because from what I saw, this fix is important.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**